blackbox/handlers/notifiers/json.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+from blackbox.handlers.notifiers._base import BlackboxNotifier
+
+
+class Json(BlackboxNotifier):
+    """A notifier for sending webhooks to a backend."""
+
+    required_fields = ("url",)
+
+    def _parse_report(self) -> dict:
+        """Turn the report into something the notify function can use."""
+        payload = []
+        for database in self.report.databases:
+            database_payload = {
+                "source": database.database_id,
+                "success": database.success,
+                "output": database.output or None
+            }
+            storages_payload = []
+            for provider in database.storages:
+                storages_payload.append({"name": provider.storage_id, "success": provider.success})
+            database_payload['backup'] = storages_payload
+            payload.append(database_payload)
```

Comment:
Slightly cramped with no newlines here, and could use a comment or two so it's easier to skim. Take a peek at the Discord notifier for an example of how this could be made a little bit more readable.

We're trying to keep this codebase extremely approachable and beginner-friendly.

Code looks good, though.

Reply:
Roger that, updated in eea20a113dedb013de50aec01607aed6abeab3da